tasks/install-certbot.yml

```diff
@@ -0,0 +1,58 @@
+---
+- name: install package dependencies.
+  package:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - "{{ dependencies }}"
+
+- name: Create the certbot group
+  group:
+    name: "{{ certbot_group }}"
+    state: present
+    system: true
+  when:
+    - certbot_group != "root"
+
+- name: Create the certbot user
+  user:
+    name: "{{ certbot_user }}"
+    groups: "{{ certbot_group }}"
+    append: true
+    shell: /usr/sbin/nologin
+    system: true
+    createhome: false
+    home: /
+  when:
+    - certbot_user != "root"
+
+- name: Create Certbot directory
+  file:
+    path: "{{ certbot_dir }}"
+    state: directory
+    owner: "{{ certbot_user }}"
+    group: "{{ certbot_group }}"
+    mode: 0755
+
+- name: Upgrade PIP and create python virtual environment for certbot
+  pip:
```

Comment:
Should be run as certbot_user

Reply:
@timothylytle Please review my comment above

Reply:
Sorry, I had put a comment futher down on the PR,  This was corrected on 2/27 with this commit
https://github.com/miarec/ansible-role-letsencrypt/pull/1/commits/957bee4d76c6aa82cc0a3cb015a8d4a69a7da5ad

I added the following to `tasks/main.yml`
```yaml
- import_tasks: install-certbot.yml
  become: true
  become_user: "{{ certbot_user }}"
```